simplified-ui-announcements/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<resources>
+  <string name="announcementTitle">%1$s (%2$d of %3$d)</string>
```

Comment:
Since these resource names aren't prefixed you should make them "private" by including a `res/values/public.xml` file with the contents:

```
<?xml version="1.0" encoding="UTF-8" ?>
<resources>
  <public />
</resources>
```

This effectively hides these resources from other modules and prevents them from showing up in autocomplete dialogs within Android Studio.

Reply:
I've been doing this in new modules but we should honestly mark all module resources private except those meant to be exported like in `ui-theme` or other `ui-` prefixed modules.

Reply:
Went ahead and did it 😅